schema.graphql

```diff
@@ -66,6 +66,42 @@ type Extrinsic @entity {
   specVersionId: Int!
 }
 
+type PermissionJson @jsonField {
```

Comment:
`PermissionValue` ?

I don't think we need Json in the name (we might have it as convention already so ignore if so).